package.json

```
@@ -23,6 +23,7 @@
   "homepage": "https://github.com/raulanatol/template-base-ts#readme",
   "devDependencies": {
     "@cowcoders/eslint-config": "1.2.11",
+    "@types/node": "^16.11.7",
```

Lo explicaremos en clase.

```suggestion
    "@types/node": "16.11.7",
```